UnityProject/Assets/Scripts/Core/Sprite Handler/SpriteHandler.cs

```diff
@@ -174,6 +186,8 @@ public void ChangeSprite(int SubCataloguePage, bool Network = true)
 				NetUpdate(NewCataloguePage: SubCataloguePage);
 			}
 		}
+
+		animateOnce = AnimateOnce;
```

Comment:
This isn't passed to net update at all ?

Reply:
ah forgot to readd it after i cleaned it up, good catch